components/gc-featured-link/gc-featured-link-en.html

```diff
@@ -1,82 +1,71 @@
 ---
-title: Featured link
-description: Documentation and working example for the featured link - provisional feature
-language: en
-altLangPage: gc-featured-link-fr.html
-dateModified: 2021-07-07
-layout: no-container
+{
+	"title": Featured link,
+	"description": Working examples for the Featured link component,
+	"language": en,
+	"altLangPage": gc-featured-link-fr.html,
+	"dateModified": 2024-03-08,
+	"breadcrumbs": [
+		{ "title": "Featured link - Documentation", "link": "components/gc-featured-link/gc-featured-link-doc-en.html" }
+	],
+	"layout": no-container
+}
 ---
+<span class="wb-prettify all-pre hide"></span>
+
 <div class="container">
-	<h1>{{ page.title }}</h1>
-
-	{% include alert-provisional.html %}
-
-	<span class="wb-prettify all-pre hide"></span>
-
-	<dl class="horizontal">
-		<dt>Status</dt>
-		<dd>Provisional</dd>
-		<dt>Type</dt>
-		<dd>Canada.ca design pattern</dd>
-		<dt>Last review</dt>
-		<dd>2021-07-07</dd>
-		<dt>Conforming to</dt>
-		<dd>Content and IA Specification - Version 2.1.2</dd>
-		<dt>Guidance</dt>
-		<dd>n/a</dd>
-		<dt>Technical name</dt>
-		<dd><code>gc-featured-link</code></dd>
-	</dl>
-
-	<h2>Purpose</h2>
-	<p>This adds a featured link to a page that your institution needs to feature prominently. It should be used to bring attention to warnings or notices.</p>
-
-	<h2>Working examples</h2>
-
-	<h3>Featured link (provisional v1.0)</h3>
-	<p>The logic semantic structure of headings have not been followed to accommodate the following working example. You can go directly to the <a href="#codeSample1">source code</a>.</p>
+	<h1 id="wb-cont" property="name">{{ page.title }}</h1>
+	<p class="mrgn-bttm-lg">Below are working examples for the "Featured link" component.</p>
+
+	<section class="alert alert-info">
```

Comment:
That margin it more because of the alert and not because of the paragraph. Preferably remove it, but if we need to keep it, please add that to the alert component instead.

```suggestion
	<p>Below are working examples for the "Featured link" component.</p>

	<section class="alert alert-info mrgn-tp-lg">
```